toolkit/tools/pkg/imagecustomizerlib/customizepackages.go

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package imagecustomizerlib
+
+import (
+	"fmt"
+	"path"
+	"strings"
+
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/imagecustomizerapi"
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/logger"
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/safechroot"
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/shell"
+)
+
+func updatePackages(buildDir string, baseConfigPath string, packagesToAddLists []string, packagesToAdd []string,
```

Comment:
Should we have an actual (maybe optional) "Update" function here that pulls the latest copies of packages?

Reply:
Done.